modules/ipi-install-establishing-communication-between-subnets.adoc

```diff
@@ -8,7 +8,12 @@
 
 In a typical {product-title} cluster setup, all nodes, including the control plane and compute nodes, reside in the same network. However, for edge computing scenarios, it can be beneficial to locate compute nodes closer to the edge. This often involves using different network segments or subnets for the remote nodes than the subnet used by the control plane and local compute nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability.
 
-Before installing {product-title}, you must configure the network properly to ensure that the edge subnets containing the remote nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too.
+Before installing {product-title}, you must configure the network properly to ensure that the edge subnets containing the remote nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too. 
+
+[IMPORTANT]
+====
+For a {product-title} cluster that is built with {op-system-base-full} and has a network that includes a bonded interface, the Network Manager might temporarily assign the wrong IP to nodes that run on network interfaces if `nodeip-configuration.service` starts before `nmstate`. This situation can cause nodes to remain stuck. If this situation occurs, reboot each node, specify the correct IP address in `/run/nodeip-configuration/primary-ip`, and restart the `wait-for-primary-ip` service.
```

Comment:
This is wrong. First of all, users don't (and mustn't) control startup order of these services, it is predefined and must stay as it is. Then, users should not be running a workaround like the one described at all. Last, but not least: Although bonded interfaces are the best example, they are not the only potential one.

What this paragraph should be saying is:
```suggestion
It is crucial that the install-time network configuration of the nodes makes them have their definitive node IP and that the nodes don't temporarily get any different node IP during their installation. As an example, if a node is to be configured with a bonded interface, their individual interfaces mustn't be assigned any different from node IP during the node setup. 
```

Reply:
Thanks, updated!